src/netstandard/pkg/shims/Directory.Build.targets

```diff
@@ -24,6 +24,12 @@
     </ItemGroup>
   </Target>
 
+  <!-- These projects have an unconventional ordering for the import of Directory.Build.Targets, so we have to manually
```

Comment:
Why don't we just delete the explicit imports of Directory.Build.Targets from the shims projects and let them be brought in by the SDK targets?

Reply:
Trying this out locally now, it requires shuffling a few things around. dotnet-install is taking forever so progress is slow.

Reply:
@ericstj I just pushed a commit to address this, I checked the artifacts dir against a version built against master, everything appears to be unchanged.